lang/python/python-s3transfer/Makefile

```diff
@@ -0,0 +1,56 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=s3transfer
+PKG_VERSION:=0.2.0
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_HASH:=f23d5cb7d862b104401d9021fc82e5fa0e0cf57b7660a1331425aab0c691d021
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/s3transfer
+PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
```

Comment:
Please rearrange as in: https://github.com/openwrt/packages/blob/master/utils/btrfs-progs/Makefile for example.

Reply:
He can rearrange as well for other python packages like docutils, awscli, botocore, colorama

Reply:
done

Reply:
Can you please look at it again? It could be lost in force-push.

Reply:
it's there now.